Source/Core/DolphinQt/Settings/GeneralPane.cpp

```diff
@@ -382,3 +366,102 @@ void GeneralPane::GenerateNewIdentity()
   message_box.exec();
 }
 #endif
+
+void GeneralPane::AddDescriptions()
+{
+  static constexpr char TR_DUALCORE_DESCRIPTION[] = QT_TR_NOOP(
+      "Runs the emulated CPU and GPU on separate threads. Increases Dolphin's frame rate, but can "
```

Comment:
I'd probably replace "frame rate" with "performance" here, as I've seen a few cases where the FPS remains the same but the game overall stutters less...